scripts/price_explorer/3_find_attribute_keys.sh

```diff
@@ -13,18 +13,21 @@ query {
     	productHash
     	attributes { key , value }
     }
-}" | jq -r '.data.products[] | @base64' | head -2
+}" | jq -r '.data.products[] | @base64'
 )
 
-i=0
-for x in $attribs; do
-    echo $x | base64 --decode | jq '.attributes | map ({ (.key): .value} ) | add' | jq ".$3"  | tee tmp-$i >/dev/null
-    i=$(expr $i + 1)
-done
-echo Found $i different products
-echo "\n#####################################\n"
+N_files=$(echo $attribs | wc -w)
 
-if [ $i -ne 1 ]; then
-     diff -U1 tmp-0 tmp-1 | grep -v "__typename"
+if [ $N_files -ne 1 ]; then
+  echo -e "\n#####################################\n"
+  echo Found $N_files different products
+  echo -e "------------------\n"
+
+  f=$(echo $attribs | cut -d' ' -f1 | base64 --decode | jq '.attributes ')
+  s=$(echo $attribs | cut -d' ' -f2 | base64 --decode | jq '.attributes ')
+
+  diff -U1 <( echo "$f" ) <( echo "$s" ) | grep -v "__typename" | grep -v "/dev/fd/"
```

Comment:
@m-yosefpor is there a chance of instead of using a `diff` the attributes, could you collect them all and present a unique list of keys (and/or key/value pairs)?

Reply:
This actually was supposed to be a simple script which has a useful output, but yeah, I think we should rewrite them with python subprocess, so that we can more easily manipulate the json files and be able to implement more complex logics easily. Bash is not handy to handle this kind of works easily (or I just don't know how to do it easily).

Reply:
I think it should be possible with a `jq` script, I'll have a test and see what's possible. Might not need to reach for python

Reply:
I got a little bit frustrated with bash/jq too but @sashman feel free to give it a shot in another PR, I copped-out in my previous scripts PR and went with `echo "$attribs" | sort | uniq` for `4_find_attribute_values.sh`